CMakeLists.txt

```diff
@@ -17,6 +17,14 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake" ${CMAKE_MODULE_PATH})
 # set the project name
 project(velox)
 
+if((NOT DEFINED ENABLE_ALL_TESTS) OR (ENABLE_ALL_TESTS MATCHES "TRUE"))
+  message(STATUS "Enabling all tests")
```

Comment:
Use `option`:
```
option(VELOX_BUILD_TESTING "Enable Velox tests" ON)
```
https://cmake.org/cmake/help/latest/command/option.html

Also IMO we should prefix the variable with VELOX_ for clarity; the primary purpose here after all is to disable testing for Velox when it is built as a subproject.

Reply:
That makes sense. I will change that